packages/protocol/contracts-0.8/UsingRegistry8.sol

```diff
@@ -4,20 +4,9 @@ pragma solidity >=0.8.0 <0.8.20;
 import "@openzeppelin/contracts8/access/Ownable.sol";
```

Comment:
Wonder if, for consistency and future-proofness, it would make sense to organize this directory with a similar structure as the original `contracts/`, i.e. put this in `contracts-0.8/common/`.

Reply:
yeah, that's a great point